pkg/controller/yurtcoordinator/podbinding/podbinding_controller.go

```diff
@@ -59,25 +62,53 @@ var (
 		Operator: corev1.TolerationOpExists,
 		Effect:   corev1.TaintEffectNoExecute,
 	}
-	defaultTolerationSeconds = 300
 )
 
+func Format(format string, args ...interface{}) string {
+	s := fmt.Sprintf(format, args...)
+	return fmt.Sprintf("%s: %s", ControllerName, s)
+}
+
 type ReconcilePodBinding struct {
 	client.Client
-	podBindingClient kubernetes.Interface
 }
 
 // Add creates a PodBingding controller and adds it to the Manager with default RBAC. The Manager will set fields on the Controller
 // and Start it when the Manager is Started.
-func Add(_ *appconfig.CompletedConfig, mgr manager.Manager) error {
-	r := &ReconcilePodBinding{}
+func Add(c *appconfig.CompletedConfig, mgr manager.Manager) error {
+	klog.Infof(Format("podbinding-controller add controller %s", controllerKind.String()))
+	return add(mgr, newReconciler(c, mgr))
+}
+
+// newReconciler returns a new reconcile.Reconciler
+func newReconciler(_ *appconfig.CompletedConfig, mgr manager.Manager) reconcile.Reconciler {
+	return &ReconcilePodBinding{
+		Client: utilclient.NewClientFromManager(mgr, ControllerName),
```

Comment:
I think we can inject `client.Client` instead of creating a new client.

```
func (r *ReconcilePodBinding) InjectClient(c client.Client) error {
	r.Client = c
	return nil
}
```